sources/packages/api/src/route-controllers/institution-locations/models/institution-location.dto.ts

```diff
@@ -120,3 +121,34 @@ export enum DesignationStatus {
    */
   NotDesignated = "Not designated",
 }
+
+export class ScholasticStandingAPIInDTO {
```

Comment:
Please add a comment mentioning that this DTO must/will be validated using the dryRun.